packages/fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -741,32 +734,6 @@ pub fn hello_world(
         assert_eq!(result.unwrap().to_string(), ":: std :: vec :: Vec < u64 >");
     }
     #[test]
-    fn test_expand_input_param_custom_type() {
```

Comment:
Why delete the test case?  Couldn't you rename the variable types as you did in other places?

Reply:
Oh no, that was by mistake (had it deleted while debugging some stuff), thanks for catching that!